lib/src/main/scala/spinal/lib/Mem.scala

```diff
@@ -133,12 +133,25 @@ class MemPimped[T <: Data](mem: Mem[T]) {
     ret.data := ClockDomain.current.withRevertedClockEdge()(mem.readSync(ret.address))
     ret
   }
+
+  def readWriteSyncPort : MemReadWritePort[T] = {
```

Comment:
We may want to add  () after the function name, as in the future we may add optional arguements (ex read during write behaviour, and other stuff)

Reply:
(same for readWriteSyncPortWithMask)